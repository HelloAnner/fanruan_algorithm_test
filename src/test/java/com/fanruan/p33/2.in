200
853
1147
3181
3307
4313
5033
5264
5558
6702
7478
8690
9527
11131
12010
12484
13791
14124
15378
15845
16267
16501
17689
18392
18455
18903
19102
19958
20435
20822
20826
20900
21103
21442
21458
21554
21897
22665
23030
23149
23314
23897
23907
25384
25516
25709
25817
26046
26458
27613
27955
28115
28447
28626
29239
29283
30078
30368
30563
30581
30797
31507
31649
31835
32141
33486
33640
34281
34818
34840
34994
35229
36410
37257
37404
37443
38036
40119
40797
41289
41492
41938
42806
42929
43862
44018
44633
44767
45317
45395
45602
45605
45925
46170
46622
46840
47366
47428
48042
48195
48493
49023
49341
49853
50611
50926
51315
51404
51784
51948
52911
52976
54120
54977
55571
55584
55788
55898
58138
58194
58372
58888
59351
59359
59380
60486
61521
62043
62247
63941
64061
64078
64332
64648
64774
65278
65605
66150
66643
67030
68363
68995
70237
70548
70601
71045
71216
71221
71431
72857
73313
73675
73932
74366
75162
75662
75930
76843
77383
77547
77764
78490
78757
79024
79095
79621
79961
81660
82422
82458
82662
83686
83756
84700
85469
86850
87095
87320
88668
90431
90814
91421
91544
92322
92512
92902
93054
93354
93461
94205
94417
95256
95718
95970
96973
97587
97660
97848
98162
98617
99871
500
15473
17195
40164
45676
52494
54339
56220
61332
86232
105499
117101
160684
209781
220775
246099
271528
286051
296678
314384
344867
419088
430225
450942
456833
476411
495845
511176
540176
544560
604673
614521
628834
638247
639080
655409
664248
675953
762919
770720
772640
787568
838043
861778
897542
902144
948541
963806
965125
971010
987091
992272
996921
997210
997400
1070107
1078666
1090766
1091613
1106231
1129046
1183903
1195712
1231790
1243097
1244197
1273123
1298971
1309679
1330890
1349218
1349683
1380954
1430861
1479552
1535816
1540717
1551079
1570271
1572938
1577082
1604961
1651549
1675738
1696164
1712381
1712610
1717780
1722107
1741331
1759774
1808823
1808876
1827761
1849588
1856539
1866057
1906277
1938153
1944172
1977658
1985718
1991311
2016662
2018011
2024597
2029330
2038999
2052718
2097632
2142203
2188566
2211202
2226672
2229172
2286577
2287034
2299825
2300626
2315363
2321034
2340726
2386306
2401824
2425452
2433139
2452922
2465944
2479349
2499834
2501876
2510832
2520747
2571365
2628295
2641372
2649878
2652864
2671301
2698664
2757535
2759412
2800375
2850315
2875477
2886106
2910780
2960003
2999086
3000629
3022058
3080993
3083044
3086406
3089359
3095726
3148083
3150968
3154952
3167334
3170095
3258002
3275425
3280106
3301402
3322200
3332571
3350979
3352345
3354750
3372292
3434719
3437136
3437670
3441875
3444761
3453215
3491026
3499689
3527414
3551811
3565092
3567417
3572169
3577490
3615567
3625684
3632004
3638782
3690015
3790489
3790529
3801950
3873526
3873897
3874157
3880795
3885668
3888474
3911763
3918873
3933038
3979109
3982612
3989658
4094139
4116368
4119683
4125233
4178103
4194892
4197738
4201639
4205896
4263187
4273548
4279437
4296102
4313688
4325215
4332471
4348720
4356401
4395885
4412653
4431449
4435966
4483356
4483742
4492530
4494625
4499365
4506244
4531107
4540540
4542175
4544806
4561785
4612377
4617116
4642581
4660556
4660956
4671737
4729685
4745163
4780731
4800179
4880392
4893237
4901825
4904444
4910614
4913304
4916003
4955754
4968884
4974030
4979598
5021395
5074387
5096610
5130809
5173548
5180517
5225136
5247657
5266922
5276614
5280437
5287244
5308651
5354555
5357991
5375323
5396959
5418514
5440452
5471315
5476514
5492513
5508532
5522082
5524692
5532778
5538829
5570426
5592899
5599430
5621506
5628471
5633062
5653248
5769100
5802581
5808473
5811347
5836055
5851834
5874300
5895040
5948993
5952901
5986289
5989157
6008162
6010398
6018184
6019836
6038182
6050861
6084654
6089289
6124504
6134811
6150227
6172939
6174835
6177818
6232953
6258626
6264889
6272802
6273789
6282804
6333262
6353579
6360011
6378512
6410461
6436258
6444887
6500951
6515352
6528205
6565001
6611789
6629872
6630360
6657651
6679776
6688994
6733574
6749576
6750568
6753527
6785068
6792354
6826089
6865497
6867486
6875375
6889177
6897230
6938013
6949236
6953668
6977201
6983608
6988858
7095581
7222671
7329433
7358208
7406996
7411033
7418215
7433354
7511201
7551299
7576614
7577306
7599767
7625743
7640863
7658846
7690509
7706152
7738018
7743324
7743480
7750955
7763897
7772298
7777890
7779075
7793540
7804056
7817373
7830589
7841732
7847034
7854419
7882690
7889415
7898263
7901679
7916629
7928291
7932165
7977719
7978260
8002811
8056646
8106208
8120724
8145271
8149774
8162513
8169595
8172834
8183447
8229939
8231253
8234025
8248009
8255891
8291681
8312813
8320308
8327603
8332048
8368291
8369420
8391938
8450480
8483935
8498085
8501029
8512148
8514496
8523715
8529431
8554991
8574111
8583728
8592635
8646377
8661686
8677991
8749524
8759496
8783257
8796503
8803658
8825013
8832375
8850997
8882418
8900015
8907780
8963453
9008350
9023908
9028576
9029513
9033138
9040706
9060344
9129702
9135966
9156612
9179892
9207596
9221130
9261540
9272920
9314938
9333279
9370508
9390260
9416638
9418945
9423814
9455287
9492905
9501714
9510381
9541144
9567737
9590409
9630857
9631127
9655983
9668476
9672003
9686419
9705261
9752323
9753458
9766903
9777729
9792540
9831541
9838401
9871096
9874982
9898638
9915646
9985008
9994533
