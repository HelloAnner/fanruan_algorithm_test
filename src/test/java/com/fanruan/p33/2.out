b021b4d2d6499d19ff90daf90365dcd4